packages/components/src/components/domainproperties/issues/actions.ts

```diff
@@ -0,0 +1,54 @@
+import {IssuesListDefModel} from "./models";
+import {ActionURL, Ajax, Domain, getServerContext, Utils} from "@labkey/api";
+
+export function fetchIssuesListDefDesign(issueDefName: string): Promise<IssuesListDefModel> {
+    return new Promise((resolve, reject) => {
+
+        Domain.getDomainDetails({
+            containerPath: getServerContext().container.path,
+            schemaName: 'issues',
+            queryName: issueDefName,
+            success: data => {
+                resolve(IssuesListDefModel.create(data));
+            },
+            failure: error => {
+                reject(error);
+            },
+        });
+    })
+
+}
+
+export function getUsersForGroup(groupId: any): Promise<any> {
+
+    return new Promise((resolve, reject) => {
+        Ajax.request({
+            url: ActionURL.buildURL('issues', 'GetUsersForGroup'),
```

Comment:
very minor, but for consistency we usually put the `.api` behind the action name:
```suggestion
            url: ActionURL.buildURL('issues', 'GetUsersForGroup.api'),
```

Reply:
updated